packages/agent/package.json

```
@@ -69,7 +69,7 @@
   "dependencies": {
     "@tbd54566975/dwn-sdk-js": "0.2.8",
     "@web5/common": "0.2.1",
-    "@web5/crypto": "0.2.2",
+    "@web5/crypto": "0.2.3",
```

`@web5/agent` still needs to be updated before using `@web5/crypto` version `0.2.3` so reverting.
Details here: https://github.com/TBD54566975/web5-js/pull/318
```suggestion
    "@web5/crypto": "0.2.2",
```